### PR TITLE
Either improves or breaks credits

### PR DESCRIPTION
**Pull request by ike709**
:cl: ike709
tweak: Credits either work much better or don't work at all. Let's find out together!
add: "Toggle Credits" has been added to the Preferences tab.
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
Please test please

**Comment by ike709** [comment]
Read discord, nerd.

**Comment by qwertyquerty** [comment]
the toggle credits verb is rather cring,

**Comment by ike709** [comment]
The verb already existed except it was added only while credits were playing. I made it a proper pref verb.

**Comment by qwertyquerty** [comment]
Correct me if I'm wrong but don't you need a new savefile version when adding prefs (one of the main reasons I wanna move prefs to the db, because I hate savefiles and savefile versions BS)

**Comment by ike709** [comment]
You are wrong. Don't need a savefile to add new things, just need a savefile to change things.

File: code/_onclick/hud/credits.dm [hunk 1]
```diff
@@ -23,39 +23,29 @@ GLOBAL_LIST(end_titles)
...
-/client/proc/ClearCredits()
-	set name = "Hide Credits"
-	set category = "OOC"
-	verbs -= /client/proc/ClearCredits
-	QDEL_LIST(credits)
+	for(var/i in 1 to GLOB.end_titles.len)
+		var/C = GLOB.end_titles[i]
+		if(!C)
+			continue
+		addtimer(CALLBACK(GLOBAL_PROC, /proc/create_credit, C), (CREDIT_SPAWN_SPEED+(i*10)), TIMER_CLIENT_TIME)
```
Comment (qwertyquerty) [inline code/_onclick/hud/credits.dm]: Shouldn't this just be `CREDIT_SPAWN_SPEED * i`

Comment (ike709) [reply code/_onclick/hud/credits.dm]: Not unless you want all of the text to stack on top of itself: https://i.imgur.com/3cgqcie.png

This is an addtimer, it doesn't pause the for loop, it just runs the code after the time has passed.

Comment (qwertyquerty) [reply code/_onclick/hud/credits.dm]: I'm aware of how it works, but doesn't multiplying it by `i` make it have an increasing time before each one spawns?

Comment (qwertyquerty) [reply code/_onclick/hud/credits.dm]: I'm not sure you did what I said right, could you show me the line of code you put in?

Comment (ike709) [reply code/_onclick/hud/credits.dm]: Use discord, this is way more tedious.

**Comment by qwertyquerty** [comment]
What could possibly go wrong

**Pull request merged**